sdk/core/azure-core/inc/azure/core/core.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+/**
+ * @file core.hpp
+ * @brief Adds all headers from Azure Core.
+ *
+ */
+
+// azure/core
+#include "azure/core/azure.hpp"
```

Comment:
azure.hpp should go away.  The contents of azure.hpp should be in a strings.hpp header which core includes.  For now until we break things azure.hpp could include that same strings header